### DISCUSSION TITLE
Updating hardhat-viem to viem 2.x

### DISCUSSION THREAD

**Discussion by JavierFerrerGomez** [NomicFoundation/hardhat#4807] [General]
I guess this is far from being critical to anybody but, there is any intention to update the plugin to use the 2.x version of Viem any time soon? Asking for a friend ;)

**Comment by schaable** [comment]
Hi @JavierFerrerGomez. There is an [issue](https://github.com/NomicFoundation/hardhat/issues/4749) for this and we're working on it. Hopefully we can have it ready soon.
